db/migrations/20200708003133_create_breeze_tables.cr

```diff
@@ -0,0 +1,47 @@
+class CreateBreezeTables::V2 < Avram::Migrator::Migration::V1
+  def migrate
+    # Learn about migrations at: https://luckyframework.org/guides/database/migrations
+    create table_for(BreezeRequest) do
```

Comment:
This is done in db/migrations for simplicity, but what we'd really want to do is have this migration class *in* Breeze. That way it automatically gets loaded and run when you do `lucky db.migrate` and have required the breeze shard.

Reply:
Note: it is possible to save all these in-memory isntead, but I opted for in-db so it is persistent. We'd want to have some kind of cleanup task or something since these could get quite big.